src/multiformats/codecs/index.ts

```diff
@@ -0,0 +1,22 @@
+import { BeeSon } from '../../beeson'
+import { JsonValue } from '../../types'
+
+export const name = 'beeson'
+
+export const code = 0xfc
+
+/**
+ * @param {BeeSon<JsonValue>} node
+ * @returns {ByteView<Uint8Array>}
+ */
+export const encode = (node: BeeSon<JsonValue>): Uint8Array => {
+  return node.serialize()
+}
+
+/**
+ * @param {ByteView<Uint8Array>} dataBytes
+ * @returns {Uint8Array}
```

Comment:
Wrong jsdoc types.
BTW types does not have to be notated here since we are using typescript. 
If you don't provide description for the params/method just leave it out.